service/src/main/java/bio/terra/profile/app/controller/ProfileApiController.java

```diff
@@ -88,10 +88,10 @@ public ResponseEntity<SamPolicyModelList> getProfilePolicies(@PathVariable("prof
   public ResponseEntity<SamPolicyModel> addProfilePolicyMember(
       @PathVariable("profileId") UUID id,
       @PathVariable("policyName") String policyName,
-      @PathVariable("memberEmail") String memberEmail) {
+      @RequestBody PolicyMemberRequest requestBody) {
```

Comment:
I'd recommend against copying down these REST annotations from the base class, as they have no benefit, only downsides. They _do_ override the values set in the base class, but aren't automatically updated when the base class version changes. So you end up with cases where a value changes in the yaml but it's not reflected in the code. The most common case is with a default value, where the value is changed in the base class interface but not changed in the subclass, so the openapi yaml change doesn't have an effect. That's why I think it's better to not annotate the subclasses at all.

Reply:
Thanks for the information @pshapiro4broad! I have removed them in https://github.com/DataBiosphere/terra-billing-profile-manager/pull/42.